weblate/trans/tests/test_reports.py

```diff
@@ -185,27 +175,45 @@ def test_counts_view_json(self) -> None:
         self.assertJSONEqual(response.content.decode(), COUNTS_DATA)
 
     def test_counts_view_30days(self) -> None:
-        response = self.get_counts("json", period="30days")
+        end = timezone.now()
+        start = timezone.now() - timedelta(days=30)
```

Comment:
Avoid race condition for different range when running testsuite on midnight (same applies to other tests):

```suggestion
        start = end - timedelta(days=30)
```

Reply:
Changed the code according to the suggested change.